tests/logictest/suites/gen/02_function/02_0012_function_datetimes

```diff
@@ -103,87 +103,72 @@ select to_datetime('0999-12-31 23:59:59');
 ----
 1000-01-01 00:00:00.000000
 
+statement query T
+select to_datetime('2022-12-31T23:59:59+00:00');
+
+----
+2022-12-31 23:59:59.000000
 
```

Comment:
how about adding some test cases for other timezones?

Reply:
Yes, in this test file:tests/logictest/suites/gen/11_data_type/11_0001_data_type_date_time